lib/core/versions/latest/models/ProtocolParameters.ts

```diff
@@ -14,6 +14,8 @@ export default interface ProtocolParameters {
   maxEncodedHashStringLength: number;
   /** Max number of operations observed per transaction time */
   maxNumberOfOpsPerTransactionTime: number;
+  /** Maximum number of operations allowed with no lock. */
+  maxNumberOfOpsForNoValueTimeLock: number;
```

Comment:
Prefer naming consistency: (not sure how`maxNumberOfOpsPerTransactionTime` fell through review cracks). Open to have the change `operations` to `ops` throughout discussion 'coz I get it gets rather wordy....
```suggestion
  maxNumberOfOperationsForNoValueTimeLock: number;
```

Reply:
Renaming.